_includes/langselect.html

```diff
@@ -1,17 +1,218 @@
-<div id="langselect">
-  <div lang="en" id="suglang">This site might also be available in your native language. Select your preferred language below:</div>
-  <ul>
-    {% for lng in site.languages %}
-      {% if lng == site.active_lang %}
-        <li class="selected" data-slang="{{ lng }}">{{ lng }}</li>
-      {% else %}
-        {% if lng == site.default_lang %}
-          {% capture langlink %} {{ page.url }}{% endcapture %}
-        {% else %}
-          {% capture langlink %}/{{ lng }}{{ page.url }}{% endcapture %}
-        {% endif %}
-        <li><a href="{{ langlink }}" data-slang="{{ lng }}">{{ lng }}</a></li>
-      {% endif %}
-    {% endfor %}
-  </ul>
+<div id="langselect-container">
+  <div id="langselect">
+    <!-- Language Dropdown -->
+    <div class="language-dropdown">
+      <button class="dropdown-button">
+        <span class="dropdown-button-content">
+          <img src="/assets/img/lang-icon.png" alt="Language Icon" class="lang-icon">
```

Comment:
Don't use absolute links here. Jekyll should support building the path somewhat dynamically.